src/contents/articles/hello-world/index.md

```diff
@@ -27,4 +25,4 @@ We want to grow as software developers but we also want to grow as a people. As
 
 ## Join us!
 
-We got a whole lot of things planned out for you, and we really believe we'll all feel better about our profession and passion if we do it together. With namespace, local developers and enthusiasts will finally have that common ground - one module, no globals.
```

Comment:
Unfortunately 'no globals' might be something you'll be seeing more often :P. I'm actually planning to make that our cheesy signature. It plays on the namespace idea, every contained within the module working together and keeping a local context!